src/pocketmine/lang/locale

```diff
@@ -1 +0,0 @@
-Subproject commit a7d53d9229337323220c7370916a153accc0ebf3
```

Comment:
So I am happy to start over (third time hopefully is the charm)!  But what I don't know how to do is clone the repo with --recursive and then due a pull request.   If I just issue  git clone --recursive https://github.com/PocketMine/PocketMine-MP from my mac - then how do I issue a pull request from my mac?

I thought I had to clone the repo to another githib repo.  But when doing that I don't have any command line access, just what the the web interface provides, which appears to be the fork command.  If I can directly clone, and then issue a pull request somehow from my mac to the https://github.com/PocketMine/PocketMine-MP repo, thats cool with me.  I am just not sure how to do that.

Thanks in advance for your help!